masterlist.yaml

```
@@ -20143,6 +20143,31 @@ plugins:
       - crc: 0xA73D91C6
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Skingrad Market( - MoC)? 1.0\.esp'
```

The first `.` should be escaped too:
```suggestion
  - name: 'Skingrad Market( - MoC)? 1\.0\.esp'
```